batch/src/main/java/org/openmrs/analytics/FhirEtlOptions.java

```diff
@@ -36,9 +36,9 @@
 	// TODO merge this with `openmrsServerUrl` to `fhirBaseUrl`
 	@Description("OpenMRS server fhir endpoint")
 	@Default.String("/ws/fhir2/R4")
-	String getServerFhirEndpoint();
+	String getOpenmrsfhirBaseEndpoint();
```

Comment:
Please add a TODO comment to remove this option completely. The only reason that this part of the URL is separated from the rest is because of the atom-feed module which is deprecated and should be removed soon.